examples/nextjs/pages/_app.js

```diff
@@ -4,15 +4,18 @@ import Head from 'next/head';
 import { ThemeProvider } from '@mui/material/styles';
 import CssBaseline from '@mui/material/CssBaseline';
 import { CacheProvider } from '@emotion/react';
-import theme from '../src/theme';
+import useDarkMode from 'use-dark-mode';
 import createEmotionCache from '../src/createEmotionCache';
+import { darkTheme, lightTheme } from '../src/theme';
 
 // Client-side cache, shared for the whole session of the user in the browser.
 const clientSideEmotionCache = createEmotionCache();
 
 export default function MyApp(props) {
+  const { value: isDark } = useDarkMode(true);
```

Comment:
```suggestion
  const { value: isDark } = useDarkMode(false);
```

Let's not change the default from light mode :)